api/v1alpha1/flowcollector_types.go

```diff
@@ -224,9 +224,29 @@ type FlowCollectorConsolePlugin struct {
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
 
+	//+kubebuilder:validation:Enum=trace;debug;info;warn;error;fatal;panic
+	//+kubebuilder:default:=info
+	// LogLevel defines the log level for the console plugin
+	LogLevel string `json:"logLevel,omitempty"`
+
 	// HPA spec of an horizontal pod autoscaler to set up for the plugin Deployment.
 	// +optional
 	HPA *FlowCollectorHPA `json:"hpa,omitempty"`
+
+	//+kubebuilder:default:={enable:true}
+	// Configuration of the port to service name translation
+	PortNaming ConsolePluginPortConfig `json:"portnaming,omitempty"`
+}
+
+// Configuration of the port to service name translation feature of the console plugin
+type ConsolePluginPortConfig struct {
+	//+kubebuilder:default:=true
+	// Should this featur be enabled
+	Enable bool `json:"enable,omitempty"`
+
+	// Additional port name to use in the console
+	// +optional
+	PortNames map[string]string `json:"portnames,omitempty"`
```

Comment:
In the description doc, could you add an example of what is expected? something like:

```
// E.g. portNames: { "9090": "open-metrics" }
```